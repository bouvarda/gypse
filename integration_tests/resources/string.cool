
class Main() {
    {
      var str: String = "Hello, world";
      out_string(str);
      out_nl();
      out_int(str.length());
      out_nl();
      out_string("Good bye, world")
    };
}