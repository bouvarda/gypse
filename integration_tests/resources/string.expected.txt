Hello, world
12
Good bye, world